e2e-tests/tests/e2e_flows.py

```diff
@@ -0,0 +1,310 @@
+    }
+
+    response_status, response_content = anonymize(json.dumps(anonymizer_request))
+
+    expected_response = (
+        """{"result":"<PERSON> drivers license is AC432223"}"""
+    )
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+
+@pytest.mark.api
+def test_given_a_correct_analyze_input_high_threashold_and_upported_entities_then_anonymize_partially():
+    language_query_parameter = "language=en"
+
+    response_status, response_content = analyzer_supported_entities(
+        language_query_parameter
+    )
+
+    assert response_status == 200
+    suppotred_entities = json.loads(response_content)
+
+    analyzer_request = {
+        "text": "John Smith drivers license is AC432223",
+        "language": "en",
+        "remove_interpretability_response": 1,
+        "score_threshold": 0.7,
+        "entities": suppotred_entities
+    }
+
+    response_status, response_content = analyze(json.dumps(analyzer_request))
+
+    expected_response = """
+    [
+        {"entity_type": "PERSON", "start": 0, "end": 10, "score": 0.85, "analysis_explanation": null}
+    ]
+    """
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+    analyzer_data = json.loads(response_content)
+
+    anonymizer_request = {
+        "text": analyzer_request["text"],
+        "transformations": {
+            "DEFAULT": {"type": "replace", "new_value": "ANONYMIZED"},
+            "US_DRIVER_LICENSE": {"type": "mask", "masking_char": "*", "chars_to_mask": 4, "from_end": True},
+            "PERSON": {"type": "replace", "new_value": "<PERSON>"}
+        },
+        "analyzer_results": analyzer_data
+    }
+
+    response_status, response_content = anonymize(json.dumps(anonymizer_request))
+
+    expected_response = (
+        """{"result":"<PERSON> drivers license is AC432223"}"""
+    )
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+
+@pytest.mark.api
+def test_given_an_unknown_entity_then_anonymize_uses_defaults():
+    analyzer_request = json.loads("""
+    {
+        "text": "John Smith drivers license is AC432223",
+        "language": "en"
+    }
+    """)
+
+    response_status, response_content = analyze(json.dumps(analyzer_request))
+
+    expected_response = """
+    [
+        {"entity_type": "PERSON", "start": 0, "end": 10, "score": 0.85, 
+        "analysis_explanation": {
+            "recognizer": "SpacyRecognizer", "pattern_name": null, "pattern": null, "original_score": 0.85, 
+            "score": 0.85, "textual_explanation": "Identified as PERSON by Spacy's Named Entity Recognition", 
+            "score_context_improvement": 0, "supportive_context_word": "", "validation_result": null 
+            }
+        },
+        {"entity_type": "US_DRIVER_LICENSE", "start": 30, "end": 38, "score": 0.6499999999999999, 
+        "analysis_explanation": {
```

Comment:
Maybe can we keep interpretability in one scenario to make the others more readable? WDYT?